learn/work/fairness-readmission/index.qmd

```diff
@@ -0,0 +1,517 @@
+toc: true
+toc-depth: 2
+include-after-body: ../../../resources.html
+bibliography: refs.bib
+---
+
+------------------------------------------------------------------------
+
+```{r}
+#| label: "setup"
+#| include: false
+#| message: false
+#| warning: false
+source(here::here("common.R"))
+saved_fits <- "learn/work/fairness-readmission/data/wflow_set_fit.rda"
+saved_final_model <- "learn/work/fairness-readmission/data/final_model.rda"
+
+if (file.exists(here::here(saved_fits))) {
+  eval_fits <- FALSE
+  # read outputs in from source:
+  load(here::here(saved_fits))
+  load(here::here(saved_final_model))
+} else {
+  eval_fits <- TRUE
+}
+```
+
+```{r}
+#| label: "load"
+#| include: false
+library(tidymodels)
+library(readmission)
+library(baguette)
+pkgs <- c("tidymodels", "readmission", "baguette")
+
+theme_set(theme_bw() + theme(legend.position = "top"))
+```
+
+In 2019, @obermeyer2019 published an analysis of predictions from a machine learning model that health care providers use to allocate resources. The model's outputs are used to recommend a patient for _high-risk care management_ programs:
+
+> These programs seek to improve the care of patients with complex health needs by providing additional resources, including greater attention from trained providers, to help ensure that care is well coordinated. Most health systems use these programs as the cornerstone of population health management efforts, and they are widely considered effective at improving outcomes and satisfaction while reducing costs. [...] Because the programs are themselves expensive---with costs going toward teams of dedicated nurses, extra primary care appointment slots, and other scarce resources---**health systems rely extensively on algorithms to identify patients who will benefit the most.**
+
+They argue in their analysis that the model in question exhibits substantial racial bias, where "Black patients assigned the same level of risk by the algorithm are sicker than White patients." In practice, this resulted in the reduction of "the number of Black patients identified for extra care by more than half."
+
+This article will demonstrate a fairness-oriented workflow for training a machine learning model to identify high-risk patients. Throughout the model development process, we'll consider the social impacts that different modeling decisions may have when such a model is deployed in context.
+
+## Setup
+
+The data we'll use in this analysis is a publicly available database containing information on `r format(nrow(readmission), big.mark = ",")` hospital stays from diabetes patients. The data comes from a study by @strack2014, where the authors model the effectiveness of a particular lab test in predicting readmission. A version of that data is available in the [readmission R package](https://simonpcouch.github.io/readmission/):
+
+```{r}
+#| label: "load-readmission"
+#| eval: false
```

Comment:
Do you mean to show the data (with the last line)? If not, I'd remove it.

Reply:
Ah, yes! [115a87b](https://github.com/tidymodels/tidymodels.org/pull/52/commits/115a87bc8ac9c045789fa29970a5091889296d16).